### ISSUE TITLE
Incorrect display capabilities returned when setting the LARGE_GRAPHIC_ONLY template

### ISSUE THREAD

**Issue by NicoleYarroch** [smartdevicelink/generic_hmi#228]
The `displayCapabilities` returned in the `SetDisplayLayout` response after setting the template to `LARGE_GRAPHIC_ONLY` is incorrect. Since `LARGE_GRAPHIC_ONLY` does not support soft buttons, the `softButtonCapabilities` key in the `displayCapabilities` should be omitted. 

```
2019-12-03 11:19:34.711227-0500 SDL_Smoke_Tests[32327:243032] [Protocol] Send RPC SetDisplayLayout (request), id: 62
{
    displayLayout = "LARGE_GRAPHIC_ONLY";
}
```

```
2019-12-03 11:19:34.724583-0500 SDL_Smoke_Tests[32327:243032] [Proxy] Message received: SetDisplayLayout (response), id: 62
{
    displayCapabilities =     {
        displayName = "GENERIC_DISPLAY";
        displayType = "SDL_GENERIC";
        graphicSupported = 1;
        imageFields =         (
                        {
                imageResolution =                 {
                    resolutionHeight = 40;
                    resolutionWidth = 40;
                };
                imageTypeSupported =                 (
                    "GRAPHIC_PNG"
                );
                name = choiceImage;
            },
                        {
                imageResolution =                 {
                    resolutionHeight = 40;
                    resolutionWidth = 40;
                };
                imageTypeSupported =                 (
                    "GRAPHIC_PNG"
                );
                name = menuIcon;
            },
                        {
                imageResolution =                 {
                    resolutionHeight = 150;
                    resolutionWidth = 150;
                };
                imageTypeSupported =                 (
                    "GRAPHIC_PNG"
                );
                name = cmdIcon;
            },
                        {
                imageResolution =                 {
                    resolutionHeight = 50;
                    resolutionWidth = 50;
                };
                imageTypeSupported =                 (
                    "GRAPHIC_PNG"
                );
                name = appIcon;
            },
                        {
                imageResolution =                 {
                    resolutionHeight = 490;
                    resolutionWidth = 915;
                };
                imageTypeSupported =                 (
                    "GRAPHIC_PNG"
                );
                name = graphic;
            },
                        {
                imageResolution =                 {
                    resolutionHeight = 225;
                    resolutionWidth = 225;
                };
                imageTypeSupported =                 (
                    "GRAPHIC_PNG"
                );
                name = alertIcon;
            }
        );
        mediaClockFormats =         (
        );
        screenParams =         {
            resolution =             {
                resolutionHeight = 675;
                resolutionWidth = 960;
            };
            touchEventAvailable =             {
                doublePressAvailable = 0;
                multiTouchAvailable = 0;
                pressAvailable = 1;
            };
        };
        templatesAvailable =         (
            DEFAULT,
            MEDIA,
            "NON-MEDIA",
            "LARGE_GRAPHIC_WITH_SOFTBUTTONS",
            "LARGE_GRAPHIC_ONLY",
            "GRAPHIC_WITH_TEXTBUTTONS",
            "TEXTBUTTONS_WITH_GRAPHIC",
            "TEXTBUTTONS_ONLY",
            "TEXT_WITH_GRAPHIC",
            "GRAPHIC_WITH_TEXT",
            "DOUBLE_GRAPHIC_WITH_SOFTBUTTONS"
        );
        textFields =         (
                        {
                characterSet = TYPE2SET;
                name = alertText1;
                rows = 1;
                width = 500;
            },
                        {
                characterSet = TYPE2SET;
                name = alertText2;
                rows = 1;
                width = 500;
            },
                        {
                characterSet = TYPE2SET;
                name = alertText3;
                rows = 1;
                width = 500;
            },
                        {
                characterSet = TYPE2SET;
                name = templateTitle;
                rows = 1;
                width = 50;
            },
                        {
                characterSet = TYPE2SET;
                name = menuName;
                rows = 1;
                width = 500;
            },
                        {
                characterSet = TYPE2SET;
                name = secondaryText;
                rows = 1;
                width = 500;
            },
                        {
                characterSet = TYPE2SET;
                name = tertiaryText;
                rows = 1;
                width = 500;
            },
                        {
                characterSet = TYPE2SET;
                name = menuTitle;
                rows = 1;
                width = 500;
            }
        );
    };
    info = "The RPC is deprecated and will be removed in a future version. The requested display layout is set to the main window. Please use `Show.templateConfiguration` instead.";
    resultCode = WARNINGS;
    softButtonCapabilities =     (
                {
            imageSupported = 1;
            longPressAvailable = 0;
            shortPressAvailable = 1;
            textSupported = 1;
            upDownAvailable = 0;
        },
                {
            imageSupported = 1;
            longPressAvailable = 0;
            shortPressAvailable = 1;
            textSupported = 1;
            upDownAvailable = 0;
        },
                {
            imageSupported = 1;
            longPressAvailable = 0;
            shortPressAvailable = 1;
            textSupported = 1;
            upDownAvailable = 0;
        },
                {
            imageSupported = 1;
            longPressAvailable = 0;
            shortPressAvailable = 1;
            textSupported = 1;
            upDownAvailable = 0;
        },
                {
            imageSupported = 1;
            longPressAvailable = 0;
            shortPressAvailable = 1;
            textSupported = 1;
            upDownAvailable = 0;
        },
                {
            imageSupported = 1;
            longPressAvailable = 0;
            shortPressAvailable = 1;
            textSupported = 1;
            upDownAvailable = 0;
        }
    );
    success = 1;
}
```

**Comment by NicoleYarroch** [comment]
I am having the same issue with the `GRAPHIC_WITH_TEXT` template

**Comment by theresalech** [comment]
Closed via #248
